.travis.yml

```diff
@@ -4,6 +4,10 @@ sudo: required
 dist: trusty
 language: generic
 
+cache:
+  directories:
+    - $HOME/.ccache
+
```

Comment:
```suggestion
cache: ccache
```
similar to https://github.com/PilzDE/pilz_industrial_motion/pull/254

Reply:
according to https://docs.travis-ci.com/user/caching/#ccache-cache

Reply:
@ct2034 the shorter form doesn't seem to work - see https://github.com/PilzDE/pilz_industrial_motion/pull/254#pullrequestreview-373562783

Reply:
that seems odd.